docs/root/configuration/listeners/network_filters/redis_proxy_filter.rst

```diff
@@ -48,8 +48,7 @@ Per command statistics
 ----------------------
 
 The Redis filter will gather statistics for commands in the
-*redis.<stat_prefix>.command.<command>.* namespace. By default latency stats are in milliseconds and can be
-changed to microseconds by setting the configuration parameter :ref:`latency_in_micros <envoy_v3_api_field_extensions.filters.network.redis_proxy.v3.RedisProxy.latency_in_micros>` to true.
+*redis.<stat_prefix>.command.<command>.* namespace. The latency stats are in microseconds.
```

Comment:
Are you sure default is microseconds? Quick read of code indicates it is milliseconds and I remember that is how implemented. Proto doc also indicates the same https://github.com/envoyproxy/envoy/blob/549164c42cae84b59154ca4c36009e408aa10b52/generated_api_shadow/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto#L192 - Has it changed recently?

Reply:
All of the stats are in milliseconds, besides upstream statistics which are microseconds.

Reply:
> besides upstream statistics which are microseconds.
which upstream stats?

Reply:
![image](https://user-images.githubusercontent.com/41201924/81292789-e3c27700-9074-11ea-8dd6-b96ee7346933.png)

Reply:
I see. Makes sense now. So we need clarify it just applies to upstream command level stats and keep the existing description for filter level stats as is?

Reply:
I believe so, yes.

Reply:
That's correct. I need to make the millis be the default, but I have a I bunch of other work to do first.

Reply:
Can we do a clarification here and we can merge? Thank you.

/wait

Reply:
I have updated the docs and put this under the setting sections.